android/app/src/main/java/org/fossasia/openevent/adapters/SessionsListAdapter.java

```diff
@@ -228,10 +228,14 @@ public void onBindViewHolder(final SessionViewHolder holder, final int position)
             default:
         }
 
-        if(session.isBookmarked()) {
+        if(session.isBookmarked() && (Color.parseColor(session.getTrack().getFontColor()) == Color.WHITE)) {
```

Comment:
No need to do this, we are using vector icons, tint the icon instead. Look at Views `setTint` method